metaflow/metadata/metadata.py

```diff
@@ -513,8 +513,29 @@ def _tags(self):
             tags.append("r_version:" + env["r_version_code"])
         return tags
 
-    def _register_code_package_metadata(self, run_id, step_name, task_id, attempt):
-        metadata = []
+    def _register_system_metadata(self, run_id, step_name, task_id, attempt):
+        env = self._environment.get_environment_info()
+        entries = {}
+        entries["runtime"] = env["runtime"]
+        entries["python_version"] = env["python_version"]
+        entries["date"] = datetime.utcnow().strftime("%Y-%m-%d")
```

Comment:
What is `date` used for? Since we're changing things anyway, maybe we could use a more descriptive name here, `task_start_utc_date` or something (or just drop it altogether). To me everything else here is self explanatory but date stands out 🤔

Reply:
(otherwise lgtm)

Reply:
Technically, we don't need the date here. It's available through the `created_at` attribute. I am in favor of dropping it altogether.

Reply:
👍 lets drop it then and merge this PR

Reply:
done